internal/io/embedio.go

```diff
@@ -0,0 +1,15 @@
+package io
+
+import (
+	"io/fs"
+	"path/filepath"
+	"strings"
+)
+
+// FSReadFile wraps fs.ReadFile supporting embedio on windows
+func FSReadFile(fsys fs.FS, name string) ([]byte, error) {
+	if filepath.Separator != '/' {
+		name = strings.ReplaceAll(name, string(filepath.Separator), "/")
```

Comment:
We should have a specific helper for this if we are going to do this in multiple places. Something like UpdatePathSeparators
Also, I'm not sure, but I believe rune-by-rune replacement is faster for ascii text in go

Reply:
I made a few tests, but I could not see ANY performance difference between strings.ReplaceAll and going through byte by byte. I didn't check the actual binary output, but my guess is that this is optimized by the compiler already.